docs/Future Development.md

```
@@ -0,0 +1,79 @@
+  CodeRabbit is an AI-powered code reviewer offering real-time, context-aware
+  feedback on pull requests, reducing manual effort in code reviews.
+slug: "/"
+---
+
+# 🚀 Future Developments
+
+We are actively iterating and refining CodeRabbit, and this is a sneak peek into the upcoming milestones and releases. We are focused on making the Applied AI review better than the median manual review. In addition to that, we are looking at a holistic user experience with various integrations and types of reviews.
+
+## 🧠 Applied AI Improvements
+
+We are actively trying to make the application of Generative AI more useful , relevant , meaningful for the coder and reviewer journey. Our immediate focus is 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)